spring-cloud-starter-stream-processor-tensorflow/src/main/java/org/springframework/cloud/stream/app/tensorflow/processor/TensorflowProcessorProperties.java

```diff
@@ -82,11 +106,27 @@ public void setModelFetchIndex(int modelFetchIndex) {
 		this.modelFetchIndex = modelFetchIndex;
 	}
 
-	public String getInputExpression() {
-		return inputExpression;
+	public Expression getExpression() {
+		return expression;
+	}
+
+	public void setExpression(Expression expression) {
+		this.expression = expression;
+	}
+
+	public OutputMode getMode() {
```

Comment:
`@NotNull`.
Not sure how Spring Boot populates when we type as `--mode=`, but we really can't allow the `null` for target logic.